### ISSUE TITLE
I got this file not found error, how to solve?

### ISSUE THREAD

**Issue by timmyhk852** [bmaltais/kohya_ss#2368]
14:01:08-974933 INFO     Start training LoRA Standard ...
14:01:08-975933 INFO     Validating lr scheduler arguments...
14:01:08-976928 INFO     Validating optimizer arguments...
14:01:08-977942 INFO     Validating model file or folder path
                         C:/Stable_Diffusion/stable-diffusion-webui/models/Stable-diffusion/ponyDiffusionV6XL_v6StartWit
                         hThisOne.safetensors existence...
14:01:08-979893 INFO     ...valid
14:01:08-979893 INFO     Validating output_dir path C:/Users/user/Desktop/kohya/kohya_ss/outputs existence...
14:01:08-981889 INFO     ...valid
14:01:08-982888 INFO     Validating train_data_dir path C:/Users/user/Desktop/kohya/kohya_ss/input_images existence...
14:01:08-983912 INFO     ...valid
14:01:08-983912 INFO     reg_data_dir not specified, skipping validation
14:01:08-984911 INFO     logging_dir not specified, skipping validation
14:01:08-985905 INFO     log_tracker_config not specified, skipping validation
14:01:08-986874 INFO     resume not specified, skipping validation
14:01:08-986874 INFO     vae not specified, skipping validation
14:01:08-987900 INFO     network_weights not specified, skipping validation
14:01:08-988898 INFO     dataset_config not specified, skipping validation
14:01:08-989894 INFO     Error: 'ai_amatsu' does not contain an underscore, skipping...
14:01:08-990891 INFO     Regulatization factor: 1
14:01:08-991899 INFO     Total steps: 0
14:01:08-992886 INFO     Train batch size: 1
14:01:08-992886 INFO     Gradient accumulation steps: 1
14:01:08-993893 INFO     Epoch: 10
14:01:08-994888 INFO     Max train steps: 1600
14:01:08-994888 INFO     stop_text_encoder_training = 0
14:01:08-995887 INFO     lr_warmup_steps = 160
14:01:08-998870 INFO     Saving training config to
                         C:/Users/user/Desktop/kohya/kohya_ss/outputs\ai_amatsu_pony_20240422-140108.json...
14:01:08-999867 INFO     Executing command: "C:\Users\user\Desktop\kohya\kohya_ss\venv\Scripts\accelerate.EXE" launch
                         --dynamo_backend no --dynamo_mode default --mixed_precision fp16 --num_processes 1
                         --num_machines 1 --num_cpu_threads_per_process 2
                         "C:/Users/user/Desktop/kohya/kohya_ss/sd-scripts/train_network.py" --config_file
                         "./outputs/tmpfilelora.toml" with shell=True
14:01:09-004857 INFO     Command executed.
2024-04-22 14:01:22 WARNING  A matching Triton is not available, some optimizations will not be enabled.  __init__.py:55
                             Error caught was: No module named 'triton'
2024-04-22 14:01:39 INFO     Loading settings from ./outputs/tmpfilelora.toml...                      train_util.py:3744
                    INFO     ./outputs/tmpfilelora                                                    train_util.py:3763
2024-04-22 14:01:39 INFO     prepare tokenizer                                                        train_util.py:4227
                    INFO     update token length: 75                                                  train_util.py:4244
                    INFO     Using DreamBooth method.                                               train_network.py:172
                    WARNING  ignore directory without repeats /                                       config_util.py:584
                             繰り返し回数のないディレクトリを無視します: ai_amatsu
                    INFO     prepare images.                                                          train_util.py:1572
                    INFO     0 train images with repeating.                                           train_util.py:1613
                    INFO     0 reg images.                                                            train_util.py:1616
                    WARNING  no regularization images / 正則化画像が見つかりませんでした              train_util.py:1621
                    INFO     [Dataset 0]                                                              config_util.py:565
                               batch_size: 1
                               resolution: (512, 512)
                               enable_bucket: False
                               network_multiplier: 1.0

                    INFO     [Dataset 0]                                                              config_util.py:571
                    INFO     loading image sizes.                                                      train_util.py:853
0it [00:00, ?it/s]
                    INFO     make buckets                                                              train_util.py:859
                    WARNING  min_bucket_reso and max_bucket_reso are ignored if bucket_no_upscale is   train_util.py:876
                             set, because bucket reso is defined by image size automatically /
                             bucket_no_upscaleが指定された場合は、bucketの解像度は画像サイズから自動計
                             算されるため、min_bucket_resoとmax_bucket_resoは無視されます
                    INFO     number of images (including repeats) /                                    train_util.py:905
                             各bucketの画像枚数（繰り返し回数を含む）
C:\Users\user\Desktop\kohya\kohya_ss\venv\lib\site-packages\numpy\core\fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
C:\Users\user\Desktop\kohya\kohya_ss\venv\lib\site-packages\numpy\core\_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
                    INFO     mean ar error (without repeats): nan                                      train_util.py:915
                    ERROR    No data found. Please verify arguments (train_data_dir must be the     train_network.py:212
                             parent of folders with images) /
                             画像がありません。引数指定を確認してください（train_data_dirには画像が
                             あるフォルダではなく、画像があるフォルダの親フォルダを指定する必要があ
                             ります）
14:01:41-579652 INFO     Training has ended.

**Comment by naufanituharish** [comment]
How did you solve the problems?
